core/services/job_subscriber.go

```diff
@@ -8,9 +8,18 @@ import (
 	"chainlink/core/store"
 	"chainlink/core/store/models"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"go.uber.org/multierr"
 )
 
+var (
+	jobSubscriptions = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
Optional: This one feels pretty left out of the `numbers` prefix party. `number` seems to only prefix counters, while gauges don't have one. Should we just have a prefix for anything prometheus? I like being able to know at a glance that it's metrics related. `jobSubscriptions` probably collides with a local variable somewhere in our code base.

`promJobSubscriptions`
`gaugeJobSubscriptions`
`metricsJobSubscriptions`